Include/cpython/object.h

```diff
@@ -167,10 +167,13 @@ typedef struct {
     objobjargproc mp_ass_subscript;
 } PyMappingMethods;
 
+typedef PySendResult (*sendfunc)(PyObject *iter, PyObject *value, PyObject **result);
+
 typedef struct {
     unaryfunc am_await;
     unaryfunc am_aiter;
     unaryfunc am_anext;
+    sendfunc am_send;
```

Comment:
Should not we add corresponding `Py_TPFLAGS_HAVE_` flag for binary compatibility?

Reply:
`typeslots.h` should be updated.

Reply:
> Should not we add corresponding Py_TPFLAGS_HAVE_ flag for binary compatibility?

Do we support extensions compiled for higher versions of Python in lower versions of Python? If I compile something for 3.10 I wouldn't expect it to work in 3.9.

Reply:
IMO we don't need a `Py_TPFLAGS_HAVE_` flag and I don't remember adding one for the `as_async` slot when we implemented PEP 492.

Reply:
But if you compile something for 3.9 it is expected to work in 3.10 and do not read uninitialized slot.

Reply:
Perhaps there is a bug in the PEP 492 implementation.

Reply:
Ahh, I see now, thanks for explaining! Yeah, in this case we do need a flag.

Reply:
> Perhaps there is a bug in the PEP 492 implementation.

Well, I don't think we need to do anything now that `as_async` has been there since 3.5. But looking back, maybe we should have added a flag.

Reply:
We used an existing slot for PEP-492, so the ABI did not change.
I don't think we necessarily need a new flag. See https://bugs.python.org/issue32388